scripts/multisig/04-publish-app-frontends.js

```diff
@@ -17,7 +17,7 @@ const { readJSON } = require('../helpers/fs')
 require('@aragon/buidler-aragon/dist/bootstrap-paths')
 
 const { generateArtifacts } = require('@aragon/buidler-aragon/dist/src/utils/artifact/generateArtifacts')
-const { uploadDirToIpfs } = require('../helpers/ipfs')
+const { uploadDirToIpfs } = require('@aragon/buidler-aragon/dist/src/utils/ipfs')
```

Comment:
Maybe remove file `helpers/ipfs.js`? It's only mentioned in `deploy-aragon-std-apps.js`,  but the usage is commented out and may be replaced by aragon import as well.

Reply:
done